polkadot/node/network/availability-recovery/src/lib.rs

```diff
@@ -666,7 +667,7 @@ impl AvailabilityRecoverySubsystem {
 							}
 						},
 						None => {
-							gum::debug!(
+							gum::trace!(
```

Comment:
Why change this level ?

Reply:
Not intended. This was annoying me at some point, but I forgot to change it back :)